src/java/neroxis/transformer/MapTransformer.java

```diff
@@ -37,6 +37,11 @@
     private boolean reverseSide;
     private float angle;
     private SymmetrySettings symmetrySettings;
+    private int resize = -1;
+    private int mapSize = -1;
+    private int oldMapSize;
+    private int shiftX = 1000000;
+    private int shiftZ = 1000000;
```

Comment:
If you are just going to default to mapSize / 2 there is no need to set the value here. Just set it using an else in the argument parsing.

Reply:
okay

Reply:
The proper mapSize isn't necessarily set at that point though.

Reply:
made flags